### PR TITLE
Project 7: Mobile Menu

### PR DESCRIPTION
**Pull request by PraisesPJMT**
## Project Description
**Project 7: Mobile Menu**
This project milestone of the portfolio website, we implemented the mobile menu using our recently acquired JavaScript knowledge.

## Feature Description

- Added JavaScript EsLint linter for the project
- Implemented hamburger button open toggle: When the user clicks (or taps) the hamburger button, the mobile menu appears.
- Implemented hamburger button close toggle: When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- Implemented navigation links and menu icon integration: When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- Implemented navigation links: When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.
- Passed all linter checks

## This Feature was Tested on the Following Browsers

- [X] Chrome
- [X] Firefox
- [X] Edge
- [ ] Internet Explorer
- [ ] Safari

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "no-param-reassign": "off",
+    "eol-last": "off",
+    "import/extensions": [ 1, {
+      "js": "always", "json": "always"
+    }]
+  },
+  "ignorePatterns": [
+    "dist/",
+    "build/"
+  ]
```
Comment (Gambit142) [inline .eslintrc.json:24]: - [ ] GitHub actions are not running for `Eslint` linters. Remember that `Eslint` ensures that we write JavaScript Codes in clean and industry-standard ways. Kindly ensure that `Eslint` linters checks on Github run and that they pass. 👍

File: README.md [hunk 2]
```diff
@@ -62,6 +62,11 @@ Here is the list of projects leading to the development of the project. In some
...
 - LinkedIn: [LinkedIn](https://linkedin.com/in/ronald-mjonono-86365988)
 
+👤 **MohamedCAkE: Contributor**
+
+- GitHub: [@githubhandle](https://github.com/MohamedCK)
+- Twitter: [@twitterhandle](https://twitter.com/MohamedCK0)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mohamed-abdulhakim-2868521b6/)
 ## 🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
```
Comment (Gambit142) [inline README.md:72]: - [ ] [Optional] The section of your `README.md` file that is displayed in the image below doesn't make the README for this project look professional. You can decide to populate it with information about those who assisted you in this project or you can remove it if you have none to appreciate. 👍 

image

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @PraisesPJMT  @MohamedCK 👋, 

Good job so far!

## Highlights
- Set up correct linters 💯 
- All linter checks passed 💯 
- The mobile menu functions properly 💯 

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider the optional suggestion I made in the comments under the review. 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md (hunk 2 above)
Comment (PraisesPJMT) [reply README.md:72]: Sure! Thanks for the observation! I just removed it temporarily until I can acknowledge a contributor.

File: .eslintrc.json (hunk 1 above)
Comment (PraisesPJMT) [reply .eslintrc.json:24]: Sure! I just realised this! I just added the Eslint GitHub auto grading!

Thanks for the observation!

**Comment by PraisesPJMT** [comment]
## Made Recommended Changes
- Add GitHub Eslint auto-grading for JavaScript Checks.

**Review by ishimwezachee — APPROVED** [review]
Hi @PraisesPJMT, @MohamedCK , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing 👍 
The project is documented professionally 👍 
Project is running and styled correctly 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**